spec/controllers/questions/start_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe Questions::StartController do
```

Comment:
Now that we have a feature spec, do we need this spec? What would happen if we switched over to using a different store for the selected answers?

Reply:
:+1: